packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -109,15 +129,22 @@ contract SystemConfig is OwnableUpgradeable, ISemver {
     /// @param data       Encoded update data.
     event ConfigUpdate(uint256 indexed version, UpdateType indexed updateType, bytes data);
 
+    /// @notice Emitted when the proposer is updated.
+    /// @param newProposer The address of the new proposer.
+    event ProposerUpdated(address indexed newProposer);
+
+    /// @notice Emitted when the challenger is updated.
+    /// @param newChallenger The address of the new challenger.
+    event ChallengerUpdated(address indexed newChallenger);
+
     /// @notice Constructs the SystemConfig contract. Cannot set
     ///         the owner to `address(0)` due to the Ownable contract's
     ///         implementation, so set it to `address(0xdEaD)`
     constructor() {
         initialize({
             _owner: address(0xdEaD),
             _superchainConfig: address(0),
-            _overhead: 0,
-            _scalar: 0,
+            _gasConfig: SystemConfig.GasConfig({ overhead: 0, scalar: 0 }),
```

Comment:
This has the `SystemConfig` prefix but the others do not

Reply:
Good call out. Fixed.